WcaOnRails/app/views/competitions/mycomps.html.erb

```diff
@@ -0,0 +1,62 @@
+<% provide(:title, 'Upcoming competitions') %>
+
+<div class="container">
+  <h2><%= yield(:title) %></h2>
+
+  <table class="table">
```

Comment:
I think this won't work so great on mobile. I'd like to look into switching this to use the `wca_table_for` I added recently for registration stuff.